src/httpCodeLensProvider.ts

```diff
@@ -39,9 +39,14 @@ export class HttpCodeLensProvider implements CodeLensProvider {
             }
 
             if (blockStart <= blockEnd) {
+                let requestVariable = null;
+                if (blockStart > 0) {
```

Comment:
This seems will break following `http` request if user has other comments after the request variable definition line:

```http
# @name login
# This is a test request!
POST https://api.example.com/login
...
```

Reply:
Rewrote the implementation so it instead checks within the delimited text scope of the request being run.